packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -765,6 +765,16 @@ class ToolbarAction extends StatelessWidget {
   }
 }
 
+class BlankHeader extends StatelessWidget implements PreferredSizeWidget {
```

Comment:
can you add some dart doc here to describe why we need a widget for this when `SizedBox()` would be the equivalent? at first glance I think it is because we need it to be a subclass of PreferredSizeWidget, but some dart doc could help